Gemfile

```
@@ -0,0 +1 @@
+source "https://rubygems.org"
```

You'd need to regenrate or ignore the Gemfile.lock too
```suggestion
source "https://rubygems.org"

gem 'github-pages'
```